pkg/provider/aci_test.go

```diff
@@ -113,6 +114,64 @@ func TestMakeRegistryCredential(t *testing.T) {
 	}
 }
 
+// Test make registry credential from docker config
+func TestMakeRegistryCredentialFromDockerConfig(t *testing.T) {
+	server := "server-" + uuid.New().String()
+	username := "user-" + uuid.New().String()
+	password := "pass-" + uuid.New().String()
+	authConfig := base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("%s:%s", username, password)))
+
+	tt := []struct {
+		name        string
+		authConfig  dockercfg.AuthConfig
+		shouldFail  bool
+		failMessage string
+	}{
+		{
+			"Valid username and password",
+			dockercfg.AuthConfig{Username: username, Password: password},
+			false,
+			"",
+		},
+		{
+			"Username and password in auth",
```

Comment:
Can you name the test more precisely? The sentence is hard to understand.

Reply:
updated name